packages/@uppy/companion/package-lock.json

```diff
@@ -0,0 +1,61 @@
+{
```

Comment:
are we also saving `package-lock` files for each uppy package? 🤔

Reply:
these package-locks only exist if lerna can't dedupe some dependencies. it seems spurious in this case though, @uppy/tus depends on the same tus-js-client version, `npm i && npm run bootstrap` may remove it again

Reply:
Removed, and then ran `npm i && npm run bootstrap`, wild `package-json.lock` didn’t re-appear 👌